clang/test/CodeGen/X86/avx10_2ni-builtins.c

```diff
@@ -0,0 +1,106 @@
+// RUN: %clang_cc1 -flax-vector-conversions=none -ffreestanding %s -triple=x86_64-unknown-unknown -target-feature +avx10.2-256 -emit-llvm -o - | FileCheck %s
```

Comment:
since generic ELF triples are the default, you can remove `-unknown-unknown` to make run lines shorter.

Reply:
Addressed in #101616